### DISCUSSION TITLE
How do I remove the divider lines on table rows?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1460] [Q&A] [answered]
Hi

I am using a table as a property bag.

![image](https://user-images.githubusercontent.com/5799306/209463133-b53878a7-7c58-4cb7-94a0-3d1b0638dc65.png)

How can I remove the divider lines on the rows?

Thank you so much.

**Comment by saadeghi** [comment, marked as answer]
One way would be adding `[&_tr>*]:border-none` class to the `table`:  
https://play.tailwindcss.com/UICShxql5h
